lib/greenhouse_api/base_client.rb

```diff
@@ -62,6 +62,15 @@ def compose_response(response)
       end
     end
 
+    def get_one(resource, id)
```

Comment:
Don't feel strongly but how do you feel about changing this to `list_one` just to stay consistent w the other methods using `list_`?

Reply:
great question. I considered that but when i thought about it, `list` and `one` felt a little oxymoronic and get one felt more like how i'd commonly ask for it. i'll keep it as is but 🤷 naming is hard haha